generators/getMetric.js

```diff
@@ -0,0 +1,90 @@
+const path = require('path')
+const fs = require('fs')
+const { metricsList } = require('./getMetricFunctions')
+const fsExtra = require('fs-extra')
+
+const CURRENCY_DESCRIPTION = `
+* @param {string} currency The metric is calculated, using a currency of choice.
+* Available currencies: USD`
+
+const TIMEBOUND_DESCRIPTION = `
+* @param {string} timeBound The metric is calculated only by taking into account the
+* tokens/coins that have moved in the past number of years or days.`
+
+function generateGetMetric () {
+  let generatedDoc = ''
+  for (const metric of metricsList) {
+    generatedDoc += allTimeBoundTemplates(metric)
+  }
+  let file = path.join(`${__dirname}`, `../src/getMetric.js`)
+  if (fs.existsSync(file)) {
+    fsExtra.removeSync(file)
+  }
+  return fs.writeFileSync(file, generatedDoc)
+}
+
+function allTimeBoundTemplates (metric) {
+  if (!('hasCurrency' in metric)) {
+    metric.hasCurrency = false
```

Comment:
can we eliminate the `if`s here?
```js
metric.hasCurrency = metric.hasCurrency || false
```
?

Reply:
This code has the same semantics but without the hard to read if not statements

Reply:
Good idea, thanks